<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>relatorios</class>
 <widget class="QWidget" name="relatorios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Relatórios</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#relatorios{
background-color: rgb(199, 199, 199);

}
QPushButton{
background-color: rgb(43, 132, 191);
color: rgb(255, 255, 255);
font-size: 15px;
border-radius: 13px;

}
QPushButton#Btn_CsvGerar{
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(43, 132, 191);
color: rgb(255, 255, 255);
font-size: 15px;
border-radius: 13px;


}

QPushButton:hover{
	background-color: rgb(13, 92, 161);
}
QPushButton:pressed{
	
color: rgb(255, 255, 255);
	background-color: rgb(153, 193, 241);
font-size: 15px;
border-radius: 13px;
}
QGroupBox{
background-color:rgb(170, 170, 170);
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 15px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Vendas</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Tipo de relatório:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="CBox_VendasMain">
             <property name="styleSheet">
              <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
             </property>
             <item>
              <property name="text">
               <string>Quantidade de vendas por período</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Valor de vendas por período</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Produtos mais vendidos</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="Stacked_Vendas">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="page">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Período:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="CBox_Periodo">
                   <item>
                    <property name="text">
                     <string>Mes</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Ano</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QComboBox" name="CBox_Ano"/>
                 </item>
                 <item>
                  <widget class="QComboBox" name="CBox_Mes"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_2">
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Período:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="CBox_periodoValor">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <item>
                    <property name="text">
                     <string>Mes</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Ano</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QComboBox" name="CBox_AnoValor"/>
                 </item>
                 <item>
                  <widget class="QComboBox" name="CBox_MesValor"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_5"/>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Estoque</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
             </property>
             <property name="text">
              <string>Tipo de Relatório:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBox_EstoqueMain">
             <property name="styleSheet">
              <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
             </property>
             <item>
              <property name="text">
               <string>Orçamento Produtos</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lista Todos Produtos</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="Stacked_Estoque">
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_3"/>
           <widget class="QWidget" name="page_4">
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>110</y>
               <width>252</width>
               <height>96</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Exporte o estoque!</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
</string>
                </property>
                <property name="text">
                 <string>Exportar estoque</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Btn_CsvGen">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Gerar CSV produtos estoque</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Btn_PdfGen">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Gerar PDF produtos estoque</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
